ignite/pkg/cosmosbuf/buf.go

```diff
@@ -167,8 +237,8 @@ func (b Buf) generateCommand(
 	return command, nil
 }
 
-// findSDKProtoPath find the cosmos-sdk proto folder path.
-func findSDKProtoPath(protoDir string) (string, error) {
+// FindSDKProtoPath finds the Cosmos SDK proto folder path.
+func FindSDKProtoPath(protoDir string) (string, error) {
```

Comment:
we don't need to export this method
```suggestion
func findSDKProtoPath(protoDir string) (string, error) {
```

Reply:
I could have sworn there was a reason I switched these to exportable but I can't remember. Since it's not necessary for this PR I reverted this change and we can revisit if I come across the requirement again